server/service/MailService.ts

```diff
@@ -43,6 +47,11 @@ interface MailContent {
   text: string;
 }
 
+type Queries =
```

Comment:
Den Namen finde ich zu generisch, dafür dass der nur in einer Methode verwendet wird.